### PR TITLE
Update/move traffic to marketing

### PR DESCRIPTION
**Pull request by belcherj**
#### Changes proposed in this Pull Request

* Moves Seo related tools from Settings->Traffic to Marketing->traffic

#### Testing instructions

* Do this with a Jetpack site, Atomic site, free plan and premium or business.
* Goto /marketing/traffic/{site}
* Verify that you can use all the features on the page
* Goto /marketing/traffic/ does this take you to the sites selector?
* Goto: settings/traffic Does it take you to site selector then marketing/traffic
* Goto settings/traffic/{site} does it redirect to /marketing/traffic/{site}

### CODE REVIEW

**Comment by robertf4** [comment]
LGTM!

**Comment by jancavan** [comment]
Design-wise, LGTM 👍

**Comment by jancavan** [comment]
@belcherj Since we have additional settings under Traffic that aren't directly related to SEO (Ads, Related Posts, etc.), @huguesvincent gave feedback that it might be best to call it Traffic. I'm in agreement with this so let's just retain the name Traffic for now.

**Comment by jancavan** [comment]
LGTM cc @huguesvincent

**Comment by huguesvincent** [comment]
Looks good to me as well.

File: client/my-sites/marketing/index.js [hunk 1]
```diff
@@ -24,13 +25,14 @@ export default function() {
 		'/marketing',
 		'/marketing/connections',
 		'/marketing/sharing-buttons',
+		'/marketing/traffic',
 		'/sharing',
 		'/sharing/buttons',
 	].forEach( path => page( path, ...[ siteSelection, sites, makeLayout, clientRender ] ) );
 
-	page( '/sharing/:domain', redirectConnections );
+	page( '/sharing/:domain', redirectTraffic );
```
Comment (stephanethomas) [inline client/my-sites/marketing/index.js]: @belcherj: why did we decide to redirect users who access the former https://wordpress.com/sharing to https://wordpress.com/marketing/traffic? We should redirect them to https://wordpress.com/marketing/connections instead in my opinion.

**Pull request merged**